source/modes/GameMode.cpp

```diff
@@ -440,12 +423,19 @@ bool GameMode::mouseMoved(const OIS::MouseEvent &arg)
 
     handleMouseWheel(arg);
 
-    handleCursorPositionUpdate();
+    if (mMouseX != inputManager->mXPos || mMouseY != inputManager->mYPos)
+    {
+        mMouseX = inputManager->mXPos;
+        mMouseY = inputManager->mYPos;
+    }
```

Comment:
You are right. This was used to avoid setting light position when it didn't move from the tile. Since now, we always update the keeper hand position, I will remove them completely